lib/mix/tasks/appsignal.demo.ex

```diff
@@ -8,5 +9,7 @@ defmodule Mix.Tasks.Appsignal.Demo do
     Appsignal.Demo.create_transaction_performance_request
     Appsignal.Demo.create_transaction_error_request
     Appsignal.stop(nil)
+    Logger.info("Demonstration sample data sent!")
+    Logger.info("It may take about a minute for the data to appear on AppSignal.com/accounts")
```

Comment:
Yes, great idea. Updated in d8b1fabdc3c9227733356df4913d8d1ebc16936f (and in [the Ruby integration](https://github.com/appsignal/appsignal-ruby/pull/235)). :+1: